h/test/admin_test.py

```diff
@@ -492,9 +506,55 @@ def test_users_index_user_found(User):
         'user_meta': {'annotations_count': 43},
     }
 
+users_delete_fixtures = pytest.mark.usefixtures('routes_mapper', 'User',
+    'delete_user')
 
-badge_index_fixtures = pytest.mark.usefixtures('models')
+@users_delete_fixtures
+def test_users_delete_redirect(User):
+    request = DummyRequest(params={"username": "bob"})
+    User.get_by_username.return_value = None
 
+    result = admin.users_delete(request)
+    assert result.__class__ == httpexceptions.HTTPFound
```

Comment:
It would be nice to test the location of the redirect as well, but it seems like the `routes_mapper` fixtures makes all `route_path` calls return a dummy path. The other solution would be to assert calls to `route_path`. Any thoughts?